samples/modelbuilder/MulticlassClassification_RestaurantViolations/RestaurantViolationsML.ConsoleApp/Program.cs

```diff
@@ -0,0 +1,49 @@
+// This file was auto-generated by ML.NET Model Builder. 
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.ML;
+using RestaurantViolationsML.Model;
+
+namespace RestaurantViolationsML.ConsoleApp
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            //await ModelBuilder.CreateModel();
+
+            // Create single instance of sample data from first line of dataset for model input
+            ModelInput sampleData = CreateSingleDataSample();
+
+            // Make a single prediction on the sample data and print results
+            ModelOutput predictionResult = ConsumeModel.Predict(sampleData);
+
+            Console.WriteLine("Using model to make single prediction -- Comparing actual Risk_category with predicted Risk_category from sample data...\n\n");
+            Console.WriteLine($"inspection_type: {sampleData.Inspection_type}");
+            Console.WriteLine($"violation_description: {sampleData.Violation_description}");
+            Console.WriteLine($"\n\nActual Risk_category: {sampleData.Risk_category} \nPredicted Risk_category value {predictionResult.Prediction} \nPredicted Risk_category scores: [{String.Join(",", predictionResult.Score)}]\n\n");
+            Console.WriteLine("=============== End of process, hit any key to finish ===============");
+            Console.ReadKey();
+        }
+
+        // Change this code to create your own sample data
+        #region CreateSingleDataSample
+        // Method to load single row of dataset to try a single prediction
+        private static ModelInput CreateSingleDataSample()
+        {
+            // Use first line of dataset as model input
```

Comment:
This comment (which is normally auto-generated) is no longer accurate since you replaced getting first line of temp file with hardcoding sample input

Reply:
#resolved